api/src/main/java/io/opencensus/trace/propagation/PropagationComponent.java

```diff
@@ -44,11 +44,22 @@
    * no implementation is provided then no-op implementation will be used.
    *
    * @since 0.11.0
-   * @return the B3 {@code TextFormat} implementation for B3.
+   * @return the B3 {@code TextFormat} implementation.
    */
   @ExperimentalApi
   public abstract TextFormat getB3Format();
 
+  /**
+   * Returns the TraceContext {@link TextFormat} with the provided implementations. See <a
```

Comment:
Add *heavy* disclaimers that this is an unstable format.. experimental api doesn't cover it, because it is the format itself which is unstable. Most people seeing something like "w3c" assume the thing is standard, not the unstable thing that changed things with very few people ack'ing this past month.